test/files/run/repl-out-dir.check

```diff
@@ -11,13 +11,17 @@ repl-out-dir-run.obj
         $eval$.class
         $eval.class
     $line2
+        $eval$$iw$$iw$.class
+        $eval$$iw$.class
```

Comment:
Yeah it does. I'm not sure what drawbacks there might be, but I avoided it for Spark-reasons.